2014/08/21 15:17:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "graphs" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('graphs')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=graphs
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 15:18:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "graphs" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('graphs')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=graphs
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 15:19:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "graphs" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('graphs')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=graphs
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 17:13:07 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'servicios.usuario' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `usuario`.
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/21 19:08:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:08:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:09:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:10:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:10:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:10:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "home2" was not found.' in C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('home2')
#1 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/servicios/index.php?r=site/page&view=home2
HTTP_REFERER=http://localhost/servicios/index.php?r=site/index
---
2014/08/21 19:16:49 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usuarios/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=usuarios/admin
HTTP_REFERER=http://localhost/servicios/index.php?r=site/login
---
2014/08/21 19:29:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "estudiantes".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('estudiantes')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/servicios/index.php?r=estudiantes
---
2014/08/21 19:49:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tutores/admin".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutores/admin')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/servicios/index.php?r=tutores/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=estudiante/admin&view=about
---
2014/08/21 19:50:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tutores/admin".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutores/admin')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/servicios/index.php?r=tutores/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=estudiante/admin&view=about
---
2014/08/21 19:50:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tutore/admin".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutore/admin')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/servicios/index.php?r=tutore/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutores/admin&view=about
---
2014/08/21 19:51:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "tutore/admin".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutore/admin')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/servicios/index.php?r=tutore/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutores/admin&view=about
---
2014/08/22 02:22:28 [error] [php] Undefined index: Password (C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php:73)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): UsuariosController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): UsuariosController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): UsuariosController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): UsuariosController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=usuarios/create
in C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php (73)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/22 02:24:03 [error] [exception.CException] exception 'CException' with message 'Property "Usuarios.Password" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('Password', 'd033e22ae348aeb...')
#1 C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php(73): CActiveRecord->__set('Password', 'd033e22ae348aeb...')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): UsuariosController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usuarios/create')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/servicios/index.php?r=usuarios/create
HTTP_REFERER=http://localhost/servicios/index.php?r=usuarios/create
---
2014/08/22 02:29:48 [error] [php] Undefined index: password (C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php:74)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): UsuariosController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): UsuariosController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): UsuariosController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): UsuariosController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=usuarios/create
in C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php (74)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/22 02:31:26 [error] [exception.CException] exception 'CException' with message 'Property "Usuarios.idUs" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('idUs')
#1 C:\xampp\htdocs\servicios\protected\controllers\UsuariosController.php(76): CActiveRecord->__get('idUs')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): UsuariosController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('usuarios/create')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/servicios/index.php?r=usuarios/create
HTTP_REFERER=http://localhost/servicios/index.php?r=usuarios/create
---
2014/08/22 02:36:00 [error] [exception.CException] exception 'CException' with message 'Property "Usuarios.Password" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('Password')
#1 C:\xampp\htdocs\servicios\protected\components\UserIdentity.php(26): CActiveRecord->__get('Password')
#2 C:\xampp\htdocs\servicios\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\servicios\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/servicios/index.php?r=site/login
HTTP_REFERER=http://localhost/servicios/index.php?r=site/login
---
2014/08/24 07:22:35 [error] [php] Undefined index: email (C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:77)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/create
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (77)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/24 07:51:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=tutor/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=site/login
---
2014/08/24 07:51:26 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=tutor/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/admin&view=about
---
2014/08/24 07:55:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('')
#1 [internal function]: TutorController->actionView('')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=
HTTP_REFERER=http://localhost/servicios/index.php
---
2014/08/24 07:55:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('')
#1 [internal function]: TutorController->actionView('')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=
HTTP_REFERER=http://localhost/servicios/index.php
---
2014/08/24 07:55:33 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=tutor/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/view&id=
---
2014/08/24 07:55:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('')
#1 [internal function]: TutorController->actionView('')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/admin&view=about
---
2014/08/24 07:56:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('')
#1 [internal function]: TutorController->actionView('')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/admin&view=about
---
2014/08/24 07:56:50 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=tutor/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/view&id=
---
2014/08/24 07:56:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('5')
#1 [internal function]: TutorController->actionView('5')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/admin&view=about
---
2014/08/24 07:57:10 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/servicios/index.php?r=tutor/admin&view=about
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/view&id=5
---
2014/08/24 07:57:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php:167
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(55): TutorController->loadModel('5')
#1 [internal function]: TutorController->actionView('5')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/admin&view=about
---
2014/08/24 08:16:41 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\components\UserIdentity.php:35)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\servicios\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=site/login
in C:\xampp\htdocs\servicios\protected\components\UserIdentity.php (35)
in C:\xampp\htdocs\servicios\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\servicios\protected\controllers\SiteController.php (94)
2014/08/24 08:21:15 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\components\UserIdentity.php:35)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\servicios\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=site/login
in C:\xampp\htdocs\servicios\protected\components\UserIdentity.php (35)
in C:\xampp\htdocs\servicios\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\servicios\protected\controllers\SiteController.php (94)
2014/08/24 08:35:35 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\components\UserIdentity.php:35)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\servicios\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=site/login
in C:\xampp\htdocs\servicios\protected\components\UserIdentity.php (35)
in C:\xampp\htdocs\servicios\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\servicios\protected\controllers\SiteController.php (94)
2014/08/25 06:56:10 [error] [exception.CException] exception 'CException' with message 'Property "Forumuser.firstseen" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('firstseen', 1408942569)
#1 C:\xampp\htdocs\servicios\protected\modules\yii-forum\components\ForumBaseController.php(27): CActiveRecord->__set('firstseen', 1408942569)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(306): ForumBaseController->beforeAction(Object(CInlineAction))
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('forum')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/servicios/index.php?r=forum
---
2014/08/27 05:29:42 [error] [exception.CException] exception 'CException' with message 'Property "Tutor.nomTutorTutor" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nomTutorTutor')
#1 C:\xampp\htdocs\servicios\protected\views\tutor\view.php(13): CActiveRecord->__get('nomTutorTutor')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(56): CController->render('view', Array)
#7 [internal function]: TutorController->actionView('5')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/view&id=5
---
2014/08/27 05:35:01 [error] [exception.CException] exception 'CException' with message 'The attribute must be specified in the format of "Name:Type:Label", where "Type" and "Label" are optional.' in C:\xampp\htdocs\yii\framework\zii\widgets\CDetailView.php:183
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CDetailView->run()
#1 C:\xampp\htdocs\servicios\protected\views\tutor\view.php(35): CBaseController->widget('zii.widgets.CDe...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(56): CController->render('view', Array)
#7 [internal function]: TutorController->actionView('5')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
HTTP_REFERER=http://localhost/servicios/index.php?r=tutor/view&id=5
---
2014/08/27 05:41:30 [error] [php] Undefined variable: form (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:15)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(56): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (15)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (56)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 05:42:33 [error] [php] include(Chtml.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 C:\xampp\htdocs\servicios\protected\views\tutor\view.php(15): spl_autoload_call()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): TutorController->renderInternal()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#5 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(56): TutorController->render()
#6 unknown(0): TutorController->actionView()
#7 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#13 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (15)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (56)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 15:25:57 [error] [php] Undefined index: secApellEstudiante (C:\xampp\htdocs\servicios\protected\controllers\EstudianteController.php:75)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): EstudianteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): EstudianteController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): EstudianteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): EstudianteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=estudiante/create
in C:\xampp\htdocs\servicios\protected\controllers\EstudianteController.php (75)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 15:26:31 [error] [php] Undefined index: secApellEstudiante (C:\xampp\htdocs\servicios\protected\controllers\EstudianteController.php:75)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): EstudianteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): EstudianteController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): EstudianteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): EstudianteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=estudiante/create
in C:\xampp\htdocs\servicios\protected\controllers\EstudianteController.php (75)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 15:26:55 [error] [exception.CException] exception 'CException' with message 'Property "Estudiante.idUsuario" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('idUsuario', '7')
#1 C:\xampp\htdocs\servicios\protected\controllers\EstudianteController.php(81): CActiveRecord->__set('idUsuario', '7')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): EstudianteController->actionCreate()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('estudiante/crea...')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/servicios/index.php?r=estudiante/create
HTTP_REFERER=http://localhost/servicios/index.php?r=estudiante/create
---
2014/08/27 19:54:41 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:26)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (26)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (58)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 19:54:57 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:26)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (26)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (58)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 19:55:40 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:27)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (27)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (58)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 19:56:06 [error] [exception.CException] exception 'CException' with message 'Property "Estudiante.nomTutor" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nomTutor')
#1 C:\xampp\htdocs\servicios\protected\views\tutor\view.php(39): CActiveRecord->__get('nomTutor')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): CController->render('view', Array)
#7 [internal function]: TutorController->actionView('5')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TutorController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TutorController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('tutor/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 C:\xampp\htdocs\servicios\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
HTTP_REFERER=http://localhost/servicios/index.php
---
2014/08/27 22:20:31 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:37)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (37)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (58)
in C:\xampp\htdocs\servicios\index.php (13)
2014/08/27 22:24:14 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\servicios\protected\views\tutor\view.php:37)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): TutorController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): TutorController->renderPartial()
#2 C:\xampp\htdocs\servicios\protected\controllers\TutorController.php(58): TutorController->render()
#3 unknown(0): TutorController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): TutorController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): TutorController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): TutorController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): TutorController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 C:\xampp\htdocs\servicios\index.php(13): CWebApplication->run()
REQUEST_URI=/servicios/index.php?r=tutor/view&id=5
in C:\xampp\htdocs\servicios\protected\views\tutor\view.php (37)
in C:\xampp\htdocs\servicios\protected\controllers\TutorController.php (58)
in C:\xampp\htdocs\servicios\index.php (13)
